Assets/MixedRealityToolkit/Utilities/Experimental/PlaneFinding/SurfaceMeshesToPlanes.cs

```diff
@@ -0,0 +1,364 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Experimental
+{
+    struct PlaneWithType
+    {
+        public GameObject Plane;
+        public SpatialAwarenessSurfaceTypes Type;
+    }
+
+    /// <summary>
+    /// SurfaceMeshesToPlanes will find and create planes based on the meshes returned by the SpatialMappingManager's Observer.
+    /// </summary>
+    public class SurfaceMeshesToPlanes:MonoBehaviour
+    {
+        [Tooltip("Empty game object used to contain all planes created by the SurfaceToPlanes class")]
+        public GameObject PlanesParent;
```

Comment:
Please convert these to [private fields with public properties (if needed)]( https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation).

This wasn't a guideline we always followed in HTK, if that's where these are coming from, but we're enforcing for MRTK v2.

Reply:
Thanks for pointing that out @keveleigh! Just updated all the properties to be private and have public accessors